examples/experimental/dagster-airlift/examples/tutorial-example/README.md

```diff
@@ -270,76 +270,77 @@ Kicking off a run of the DAG in Airflow, you should see the newly created assets
 
 _Note: There will be some delay between task completion and assets materializing in Dagster, managed by the sensor. This sensor runs every 30 seconds by default (you can reduce down to one second via the `minimum_interval_seconds` argument to `sensor`), so there will be some delay._
 
-## Migrating Assets
+## Proxying to Assets
 
-Once you have created corresponding definitions in Dagster to your Airflow tasks, you can begin to selectively migrate execution of some or all of these assets to Dagster.
+Once you have created corresponding definitions in Dagster to your Airflow tasks, you can begin to selectively _proxy_ execution of some or all of these assets to Dagster. Airlift provides utilities for 
+remotely kicking off materializations of dagster assets rather than execute the task within airflow.
 
-To begin migration on a DAG, first you will need a file to track migration progress. In your Airflow DAG directory, create a `migration_state` folder, and in it create a yaml file with the same name as your DAG. The included example at [`airflow_dags/migration_state`](./tutorial_example/airflow_dags/migration_state) is used by `make airflow_run`, and can be used as a template for your own migration state files.
+To begin proxying tasks to dagster, you'll need to track proxied state within your airflow repository. In your Airflow DAG directory, create a `proxied_state` folder, and in it create a yaml file with the same name as your DAG. The included example at [`airflow_dags/proxied_state`](./tutorial_example/airflow_dags/proxied_state) is used by `make airflow_run`, and can be used as a template for your own proxied state files.
 
-Given our example DAG `rebuild_customers_list` with three tasks, `load_raw_customers`, `run_dbt_model`, and `export_customers`, [`migration_state/rebuild_customers_list.yaml`](./tutorial_example/airflow_dags/migration_state/rebuild_customers_list.yaml) should look like the following:
+Given our example DAG `rebuild_customers_list` with three tasks, `load_raw_customers`, `run_dbt_model`, and `export_customers`, [`proxied_state/rebuild_customers_list.yaml`](./tutorial_example/airflow_dags/proxied_state/rebuild_customers_list.yaml) should look like the following:
 
 ```yaml
-# tutorial_example/airflow_dags/migration_state/rebuild_customers_list.yaml
+# tutorial_example/airflow_dags/proxied_state/rebuild_customers_list.yaml
 tasks:
   - id: load_raw_customers
-    migrated: False
+    proxied: False
   - id: build_dbt_models
-    migrated: False
+    proxied: False
   - id: export_customers
-    migrated: False
+    proxied: False
```

Comment:
I think this is great and makes it much more clear what is going on. This is tracking a specific technical behavior, not arbitrary metadata for bookkeeping.